_posts/2024-05-12-update.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Week Ending May 12, 2024
+date: 2024-05-13 22:00:00 -0000
+slug: 2024-05-12-update
+---
+
+## Developer News
+
+The Kubernetes documentation is now available in 15 different languages. Thanks to every community member who helped on the translation and launch of the
```

Comment:
```suggestion
The Kubernetes documentation is now available in 16 different languages. Thanks to every community member who helped on the translation and launch of the
```

There are 15½ localization projects (Portuguese might split into European Portuguese and Brazilian Portuguese), but English is also a language, so 16.